specs/phase0/p2p-interface.md

```diff
@@ -245,7 +245,7 @@ The following gossipsub [parameters](https://github.com/libp2p/specs/blob/master
 - `fanout_ttl` (ttl for fanout maps for topics we are not subscribed to but have published to, seconds): 60
 - `mcache_len` (number of windows to retain full messages in cache for `IWANT` responses): 6
 - `mcache_gossip` (number of windows to gossip about): 3
-- `seen_ttl` (number of heartbeat intervals to retain message IDs): 550
+- `seen_ttl` (expiry time for cache of seen message ids, seconds): SECONDS_PER_SLOT * SLOTS_PER_EPOCH * 2
```

Comment:
Why not just set it at 1100 heartbeat intervals ? The previous value was estimated using the time for 1 epoch, so if we want to persist for 2 epochs, we can just double the current value.

Reply:
The [gossipsub specification](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.0.md#parameters) suggests this TTL should be expressed in time units, not heartbeats.

Reply:
Sounds good, would be fine at setting it in terms of consensus params in that case.

Reply:
To me seconds is more sensible to express this in anyway...